src/test/java/org/assertj/core/api/shortarray/ShortArrayAssertNullTest.java

```diff
@@ -0,0 +1,28 @@
+package org.assertj.core.api.shortarray;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.assertj.core.api.AbstractShortArrayAssert;
+import org.assertj.core.api.ShortArrayAssertBaseTest;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Adds a test to verify null behavior against empty arrays.
+ * 
+ * Should be used for methods with an int[] or int... parameter.
+ * 
+ * @author Dan Avila
+ */
+public abstract class ShortArrayAssertNullTest extends ShortArrayAssertBaseTest {
+
+  @Test
+  public void should_throw_exception_on_null_argument() {
+    assertThatExceptionOfType(NullPointerException.class).isThrownBy(() -> {
+      int[] content = null;

```

Comment:
Let's use `catchThrowableOfType` and the GIVEN WHEN THEN pattern to make the test easier to read.  
shoud look like:
```java
    // GIVEN
    int[] nullContent = null;
    // WHEN
    NullPointerException npe = catchThrowableOfType(() -> invoke_api_with_null_value(assertThat(new short[] {}), nullContent),
                                                    NullPointerException.class);
    // THEN
    then(npe).hasMessage("The array of values to look for should not be null");
```